### PR TITLE
Bump to PM5

### PR DESCRIPTION
**Pull request by IvanCraft623**
The plugin should work correctly for PM4 and PM5.

### CODE REVIEW

**Comment by dktapps** [comment]
the virion config should also be updated

**Comment by IvanCraft623** [comment]
done

**Review by PJZ9n — COMMENTED** [review]
it is not the version that changes, but the api?

File: virion.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: libFormAPI
 author: jojoe77777
 api: 4.0.0
-version: 2.1.1
+version: [4.0.0, 5.0.0]
```
Comment (dktapps) [inline virion.yml]: This is the wrong field, it's `api` that should be modified (good catch @PJZ9n )

Comment (IvanCraft623) [reply virion.yml]: hahaha, I can't believe I made this

**Pull request merged**